### ISSUE TITLE
Parsing failure if missing charge column

### ISSUE THREAD

**Issue by danieleongari** [danieleongari/egulp#6]
If the `_atom_site_charge` column is missing from the CIF, the program fails with the error:
```
egulp version 2.0.2
number of command line parameters is 4
input parameters are assumed in order
egulp <geometry> <parameters> <configure>
opening geometry file HKUST1.cif
geometry file HKUST1.cif is open
Lattice lenghts: A 26.3430000 B 26.3430000 C 26.3430000
Lattice angles: alpha 90.0000000 beta 90.0000000 gamma 90.0000000
Lattice vector #1 26.3430000  0.0000000  0.0000000
Lattice vector #2  0.0000000 26.3430000  0.0000000
Lattice vector #3  0.0000000  0.0000000 26.3430000
unit cell volume is 18280.8209756
Number of atoms is 624
Arrays for atoms allocated
Rewind file HKUST1.cif to read the atoms after line: _atom_site_fract_z

Reading atoms
[1]    1384 segmentation fault (core dumped)  ../../egulp HKUST1.cif GMP.param configure.input
```

Note that I changed `manage_crystal` default settings not to print the charges column if all charges are zero, which is the reason why this failure was not relevant before.

**Comment by danieleongari** [comment]
After further inspection, there are also problems with extra spaces at the end of the lines.
I'm not sure anymore if the parsing problem is related only to this new problem or also the changes column

**Comment by ltalirz** [comment]
> problems with extra spaces at the end of the lines.

should be fixed by ae4f54a4e7d81dc9c2ec50463e89fd78101c7f77
